lib/proxy/log.rb

```diff
@@ -10,13 +17,27 @@ def logger
       @@logger ||= ::Proxy::Log.logger
     end
 
+    def self.default_logger(log_file)
+      # We keep the last 6 10MB log files
+      ::Logger.new(log_file, 6, 1024*1024*10)
+    end
+
     def self.logger
-      log_file = ::Proxy::SETTINGS.log_file
+      log_file = ENV['SMART_PROXY_LOGFILE'] || ::Proxy::SETTINGS.log_file
```

Comment:
Do we need environment override in production (I don't think so)? In which case, could you move this initialization to https://github.com/theforeman/smart-proxy/blob/develop/lib/smart_proxy_for_testing.rb#L20 ?